site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -1,25 +1,30 @@
 <div class="content">
+    <input type="hidden" id="csrf_token" value="{{ csrf_token }}"/>
+    <input type="hidden" id="base_url" value="{{ base_url }}"/>
```

Comment:
is this needed? If all of your JS in also within twig files then you can use the same `"{{ base_url }}"` to inject the value you need into the JS code, remember the quotes if you do do this to make sure the twig replacement expands to valid JS code.